### PR TITLE
style(pre-commit): apply the init pre-commit fixup

### PR DESCRIPTION
**Pull request by tai271828**
## Types of changes

- **Refactoring**

## Description

Follow the discussion of this thread https://github.com/dannf/ubuntu-server-netboot/pull/15#issuecomment-831274714 Let's add pre-commit and apply bllack fix.

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [ ] Update the documentation if necessary

## Steps to Test This Pull Request

The same as the testing method from Follow the discussion of this thread https://github.com/dannf/ubuntu-server-netboot/pull/15#issuecomment-831274714 Let's add pre-commit and apply bllack fix.

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[flake8]
+# Line break occurred before a binary operator (W503)
+# The recent research unearthed recommendations by Donald Knuth to break before
+# binary operators. flake8 will soon follow up the new recommendation. Until
+# the flake8 update, let's ignore the check.
+#
+# https://www.flake8rules.com/rules/W503.html
+extend-ignore = W503, E203
+
+max-line-length = 119
```
Comment (dannf) [inline .flake8:10]: What's the reason behind the max-line-length?

**Review by dannf — COMMENTED** [review]
Thanks @tai271828 ! This will really help consistency!
Would you mind mentioning `black` and `flake8` in the commit message? It currently just mentions the use of `pre-commit`, which is kinda burying the lead :) And just one question inline.

**Comment by tai271828** [comment]
> Thanks @tai271828 ! This will really help consistency!
> Would you mind mentioning `black` and `flake8` in the commit message? It currently just mentions the use of `pre-commit`, which is kinda burying the lead :) And just one question inline.

uh, sounds good. Will do!

File: .flake8 (hunk 1 above)
Comment (tai271828) [reply .flake8:10]: > What's the reason behind the max-line-length?

The idea is inspired by [the coding style of `Django` project](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/):

```
We allow up to 119 characters as this is the width of GitHub code review; anything longer requires horizontal scrolling which makes review more difficult.
```


My own user experience of `max-line-length = 119` is also good with modern wide monitor screen. Maybe let me quote the above statement of Django in our commit message (I think I did but maybe it was dropped accidentally when rebasing my history tree).

**Comment by tai271828** [comment]
I have updated the commit message according to the above discussion/suggestion, and this pull request is ready to review again.

File: .flake8 (hunk 1 above)
Comment (tai271828) [reply .flake8:10]: Please feel free to let me know which line wrap number you prefer. 72, 79 or 119 all works for me.

Comment (dannf) [reply .flake8:10]: Ah, OK. I tend to use 79, but I'm fine with 119.

**Review by dannf — APPROVED** [review]

**Pull request merged**